aggregator/src/main/resources/application.properties

```diff
@@ -16,6 +16,7 @@ mp.messaging.outgoing.egress.connector=smallrye-kafka
 mp.messaging.outgoing.egress.topic=platform.notifications.ingress
 mp.messaging.outgoing.egress.key.serializer=org.apache.kafka.common.serialization.StringSerializer
 mp.messaging.outgoing.egress.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+feature-flags.expose-kafka-ssl-config-keys.enabled=true
```

Comment:
Adding this configuration key allows `clowder-quarkus-config-sources` module to load kafka properties directly, without neading our additionnal `KafkaSaslInitializer` file.